css/gallery.css

```diff
@@ -0,0 +1,116 @@
+/*class for the section of gallery*/
+.info-gallery {
+    width: 635px;
+    height: 88px;
+    font-family: Roboto;
+    font-size: 16px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: black;   
+}
+.info-gallery__text {
+    top: 0;
+    left: 0;
+    margin-right: 150px;
+    margin-left: 150px;
+    margin-bottom: 17px;
```

Comment:
I think it's better to use **position: absolute** for this text.
Please, if I'm wrong, correct me.

Reply:
using absolute would imply a fixed position on reference to at-least one side, making it highly probable to be misaligned on very big or very small screens.